packages/driver/src/cypress/stack_utils.js

```diff
@@ -200,6 +199,11 @@ const parseLine = (line) => {
 }
 
 const stripCustomProtocol = (filePath) => {
+  // sometimes stack frames won't include a filepath
+  if (!filePath) {
```

Comment:
Can you add a test for this? It fixes this issue, right? https://github.com/cypress-io/cypress/issues/7870

Reply:
no, that issue is caused by throwing a string literal, it causes a different error than the one I saw. I'll revert this since we have no open issues for it